src/NavLink.php

```diff
@@ -152,6 +293,23 @@ public function label(string|Stringable $value): self
         return $new;
     }
 
+    /**
+     * Sets the HTML attributes for the tab pane.
+     *
+     * @param array $values Attribute values indexed by attribute names.
+     *
+     * @return self New instance with the specified pane attributes.
+     *
+     * @see \Yiisoft\Html\Html::renderTagAttributes() for details on how attributes are rendered.
+     */
+    public function paneAttributes(array $values): self
```

Comment:
```suggestion
    public function paneAttributes(array $attributes): self
```

Reply:
When it is a single argument, use `value` or `values`.